src/renderer/page/settings/view.jsx

```diff
@@ -1,352 +1,322 @@
-            <FormRow
-              type="checkbox"
-              onChange={this.onShowUnavailableChange.bind(this)}
-              defaultChecked={showUnavailable}
-              label={__('Show unavailable content in search results')}
-            />
-            <FormRow
-              label={__('Show NSFW content')}
-              type="checkbox"
-              onChange={this.onShowNsfwChange.bind(this)}
-              defaultChecked={showNsfw}
-              helper={__(
-                'NSFW content may include nudity, intense sexuality, profanity, or other adult content. By displaying NSFW content, you are affirming you are of legal age to view mature content in your country or jurisdiction.  '
-              )}
-            />
-          </div>
-        </section>
-
-        <section className="card">
-          <div className="card__content">
-            <h4>{__('Share Diagnostic Data')}</h4>
-          </div>
-          <div className="card__content">
-            <FormRow
-              type="checkbox"
-              onChange={this.onShareDataChange.bind(this)}
-              defaultChecked={daemonSettings.share_usage_data}
-              label={__('Help make LBRY better by contributing diagnostic data about my usage')}
-            />
-          </div>
-        </section>
-
-        <section className="card">
-          <div className="card__content">
-            <h4>{__('Theme')}</h4>
-          </div>
-          <div className="card__content">
-            <FormField
-              type="select"
-              onChange={this.onThemeChange.bind(this)}
-              defaultValue={theme}
-              className="form-field__input--inline"
-            >
-              {themes.map((theme, index) => (
-                <option key={theme} value={theme}>
-                  {theme}
-                </option>
-              ))}
-            </FormField>
-
-            <FormRow
-              type="checkbox"
-              onChange={this.onAutomaticDarkModeChange.bind(this)}
-              defaultChecked={automaticDarkModeEnabled}
-              label={__('Automatic dark mode (9pm to 8am)')}
-            />
-          </div>
-        </section>
-
-        <section className="card">
-          <div className="card__content">
-            <h4>{__('Application Cache')}</h4>
-          </div>
-          <div className="card__content">
-            <p>
-              <Link
-                label={this.state.clearingCache ? __('Clearing') : __('Clear the cache')}
-                icon="icon-trash"
-                button="alt"
-                onClick={this.clearCache.bind(this)}
-                disabled={this.state.clearingCache}
+              </div>
+            </section>
+
+            <section className="card card--section">
+              <div className="card__title">{__('Share Diagnostic Data')}</div>
+              <div className="card__subtitle">{__('List what we are doing with the data here')}</div>
```

Comment:
I would like to add exactly what data we are collecting and what we are doing with it. Not sure what that is